demos/JWST/S5_template.ecf

```diff
@@ -9,15 +9,17 @@ rescale_err     False                   # Rescale uncertainties to have reduced
 fit_par         ./S5_fit_par_template.epf   # What fitting epf do you want to use?
 verbose         True                    # If True, more details will be printed about steps
 fit_method      [dynesty]               #options are: lsq, emcee, dynesty (can list multiple types separated by commas)
-run_myfuncs     [batman_tr,polynomial]  # options are: batman_tr, batman_ecl, sinusoid_pc, expramp, polynomial, step, xpos, ypos, xwidth, ywidth, and GP (can list multiple types separated by commas)
+run_myfuncs     [batman_tr,polynomial]  # options are: batman_tr, batman_ecl, sinusoid_pc, poet_tr, poet_ecl, poet_pc, expramp, polynomial, step, xpos, ypos, xwidth, ywidth, lorentzian, damped_osc, and GP (can list multiple types separated by commas)
 compute_ltt     False                   # options are: True (correct model for the light travel time effect), or False (ignore the light travel time effect)
+force_positivity False                  # Optional boolean for sinusoid_pc and poet_pc models.  Set True to force positive phase variations.
+num_planets     1                       # Number of planets in the fit. Default is 1.
```

Comment:
Don't forget to add these to the S5 template in docs/media (and update any other files in that folder as needed)